src/main/java/org/broadinstitute/hellbender/utils/genotyper/AlleleLikelihoods.java

```diff
@@ -1180,9 +1169,12 @@ private void removeEvidenceByIndex(final int sampleIndex, final int[] evidencesT
                 }
             }
         }
+        // set to NaN lks of the deleted positions in lk value arrays.
```

Comment:
I'm proactively getting rid of the zombie lks when evidence is removed. I set them to NaN.